recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb

```diff
@@ -23,11 +25,14 @@ S = "${WORKDIR}"
 
 DOMD_CONFIG_salvator-x-m3-xt = "domd-salvator-x-m3.cfg"
 DOMD_CONFIG_salvator-x-h3-xt = "domd-salvator-x-h3.cfg"
+DOMD_CONFIG_salvator-x-m3n-xt = "domd-salvator-x-m3n.cfg"
 DOMU_CONFIG_salvator-x-m3-xt = "domu-salvator-x-m3.cfg"
 DOMU_CONFIG_salvator-x-h3-xt = "domu-salvator-x-h3.cfg"
+DOMU_CONFIG_salvator-x-m3n-xt = "domu-salvator-x-m3n.cfg"
 
 DOM0_ALLOWED_PCPUS_salvator-x-m3-xt = "2-5"
 DOM0_ALLOWED_PCPUS_salvator-x-h3-xt = "4-7"
+DOM0_ALLOWED_PCPUS_salvator-x-m3n-xt = "4-7"
```

Comment:
Please drop this. M3N doesn't have LITTLE cores we might want several domains to pin to. I even think, that you can drop this for other machines in Gen3 test.
Also please drop everything related to dom0_vcpu_pin.sh.

Reply:
Done